Kickstarter-iOS/DataSources/CommentRepliesDataSource.swift

```diff
@@ -0,0 +1,28 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+internal final class CommentRepliesDataSource: ValueCellDataSource {
+  internal enum Section: Int {
+    case comment
+    case replies
+  }
+
+  internal func load(comment: Comment, project: Project) {
+    let section = Section.comment.rawValue
+    self.clearValues()
+
+    self
+      .appendRow(value: (comment, project), cellClass: CommentCell.self, toSection: section)
```

Comment:
Not sure if this indent was intentional or not but it seems a bit weird lol

Reply:
Yes this is a big one, I'm correcting it now.